tests/test_many_scores/test_many_scores.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+
+import os
+import sys
+from subprocess import Popen, STDOUT, PIPE, call
+import filecmp
+import glob
+from optparse import OptionParser
+
+parser = OptionParser()
+parser.add_option('--mpi_exec', dest='mpi_exec', default='')
+parser.add_option('--mpi_np', dest='mpi_np', default='3')
+parser.add_option('--exe', dest='exe')
+(opts, args) = parser.parse_args()
+cwd = os.getcwd()
+
+def test_run():
+    if opts.mpi_exec != '':
+        proc = Popen([opts.mpi_exec, '-np', opts.mpi_np, opts.exe, cwd],
+               stderr=STDOUT, stdout=PIPE)
+    else:
+        proc = Popen([opts.exe, cwd], stderr=STDOUT, stdout=PIPE)
+    print(proc.communicate()[0])
+    returncode = proc.returncode
+    assert returncode == 0, 'OpenMC did not exit successfully.'
+
+def test_created_statepoint():
+    statepoint = glob.glob(os.path.join(cwd, 'statepoint.10.*'))
+    assert len(statepoint) == 1, 'Either multiple or no statepoint files exist.'
+    assert statepoint[0].endswith('binary') or statepoint[0].endswith('h5'),\
+        'Statepoint file is not a binary or hdf5 file.'
+
+def test_output_exists():
+    assert os.path.exists(os.path.join(cwd, 'tallies.out')), 'Tally output file does not exist.'
+
+def test_results():
+    statepoint = glob.glob(os.path.join(cwd, 'statepoint.10.*'))
+    call([sys.executable, 'results.py', statepoint[0]])
+    compare = filecmp.cmp('results_test.dat', 'results_true.dat')
+    if not compare:
+      os.rename('results_test.dat', 'results_error.dat')
```

Comment:
Should be indented two more spaces. I realize this is copy/paste, so the wrong indentation is elsewhere too.